find_me_a_sandwich/features/step_definitions/make_favorites_steps.rb

```diff
@@ -0,0 +1,25 @@
+Given(/^I am looking at a merchant with a menu$/) do
```

Comment:
I might make one file for all Favorites steps.  You don't need to make step definitions 1-1 with their features.